### DISCUSSION TITLE
Chromecast to Airplay ?

### DISCUSSION THREAD

**Discussion by dr3amr** [hassio-addons/app-aircast#179] [Ideas]
Could the other way round be achieved ? I'm looking to have virtual Chromecast devices for all Airplay devices in network (mainly my Yamaha Receiver, but perhaps also future AppleTV devices). Possibly this can be done through a new addon. But interested if it is possible.

**Comment by frenck** [comment]
> Could the other way round be achieved ? 

Probably, but not what this add-on is about.
